pandas/core/frame.py

```diff
@@ -7008,7 +7008,6 @@ def unstack(self, level=-1, fill_value=None):
              b   4.0
         dtype: float64
 
-        >>> s.unstack(level=-1)
```

Comment:
@Rik-de-Kort I don't think you intended to remove this.

 the doc error is `/home/runner/work/pandas/pandas/pandas/core/frame.py:docstring of pandas.DataFrame.unstack:55: WARNING: Block quote ends without a blank line; unexpected unindent.`

so could be related.

Reply:
ahhh thanks! all checks passing now, finally!